### DISCUSSION TITLE
无法安装任何Forge版本

### DISCUSSION THREAD

**Discussion by TidalStarNan** [Meloong-Git/PCL#2560] [提问] [answered]
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/12406520/Log1.txt)
PCL为正式版2.6.6，Windows11

下载 Forge 47.1.44失败
→ 获取 Mod 下载器支持库文件失败
→ 设备未就绪。

   在 PCL.ModDownloadLib._Closure$__37-1._Lambda$__3(LoaderTask`2 Task)
   在 PCL.ModLoader.LoaderTask`2._Closure$__12-0._Lambda$__0()
错误类型：System.IO.IOException

**Comment by zkitefly** [comment]
上传启动器日志

**Reply by TidalStarNan** [reply]
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/12403748/Log1.txt)

**Reply by LTCatt** [reply, marked as answer]
应该是系统或者硬盘问题，不是启动器导致的 :/

**Comment by SuoHouDaiShi** [comment]
<del>Win11的话等12出来再升级吧，先用10好了（bushi</del>
